_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,283 @@
+---
+id: bc4a75
+name: ARIA required owned elements
+rule_type: atomic
+description: |
+ This rule checks that each role has at least one of its required owned elements.
+accessibility_requirements:
+ wcag20:1.3.1: # Info and Relationships (A)
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [WAI-ARIA required owned elements](https://www.w3.org/TR/wai-aria/#mustContain), except when the owned element has an implicit semantic role that is identical to its explicit semantic role.
```

Comment:
I do not really understand:
* why we only want to check element with explicit roles, not those with implicit roles.
* why there is the exception at the end of the applicability clause.

(that's more curiosity than real comment)

Reply:
Thanks for the above 2 comments. I was trying to line up everything similar to #255 and that caused some ambiguity. I am updating this to simple language.